src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -944,8 +970,6 @@ private static byte HexDigitToBinary(char c)
         {
             if ('0' <= c && c <= '9')
                 return (byte)(c - '0');
-            if ('A' <= c && c <= 'F')
-                return (byte)(c - ('A' - 10));
```

Comment:
Is this being removed because the spec guarantees IDs will use lower-case hex letters?  I'm wondering if it's still worthwhile to keep, even with that.

Reply:
spec requires lowercase and we should treat uppercase as invalid IDs

Reply:
I am OK with following the spec until we know the value of relaxing our input criteria.

Reply:
Ok